lib/workers/branch/schedule.spec.ts

```diff
@@ -150,11 +159,53 @@ describe('workers/branch/schedule', () => {
       const res = schedule.isScheduledNow(config);
       expect(res).toBeFalse();
     });
+
     it('supports outside hours', () => {
       config.schedule = ['after 4:00pm'];
       const res = schedule.isScheduledNow(config);
       expect(res).toBeFalse();
     });
+
+    it('supports cron syntax with hours', () => {
+      config.schedule = ['* 10 * * *'];
+      let res = schedule.isScheduledNow(config);
```

Comment:
Do we already mock current time, so theses tests work?

Reply:
@viceice yes, I think so this block of code does the date mocking:

```
beforeEach(() => {
      mockDate.set('2017-06-30T10:50:00.000'); // Locally 2017-06-30 10:50am
      jest.resetAllMocks();
      config = {};
    });
```